src/Analysis/GWASV2/Shared/CohortsOverlapDiagram/CohortsOverlapDiagram.stories.jsx

```diff
@@ -0,0 +1,132 @@
+import React, { useState } from 'react';
+import { QueryClient, QueryClientProvider } from 'react-query';
+import { rest } from 'msw';
+import CohortsOverlapDiagram from './CohortsOverlapDiagram';
+
+export default {
+  title: "Tests3/GWASV2/CohortsOverlapDiagram",
+  component: CohortsOverlapDiagram,
+};
+
+const mockedQueryClient = new QueryClient({
+  defaultOptions: {
+    queries: { retry: false },
+  },
+});
+
+const Template = (args) =>
+<QueryClientProvider client={mockedQueryClient}>
+  <CohortsOverlapDiagram {...args} />
+</QueryClientProvider>;
+
+
+const selectedStudyPopulationCohort = {
+  cohort_definition_id: 25,
+  cohort_name: 'Mock population ',
+  size: 4000,
+};
+const selectedCaseCohort = {
+  cohort_definition_id: 123,
+  cohort_name: 'Mock case cohort A',
+  size: 2500,
+};
+const selectedControlCohort = {
+  cohort_definition_id: 456,
+  cohort_name: 'Mock control cohort  with loooong name to test how a long name is rendered',
+  size: 1800,
+};
+
+export const SuccessCase = Template.bind({});
+SuccessCase.args = {
+  sourceId: 123,
+  selectedStudyPopulationCohort: selectedStudyPopulationCohort,
+  selectedCaseCohort: selectedCaseCohort,
+  selectedControlCohort: selectedControlCohort,
+  selectedCovariates: [],
+  selectedDichotomousCovariates: [],
+}
+SuccessCase.parameters = {
+  // msw mocking:
+  msw: {
+    handlers: [
+      rest.post('http://:cohortmiddlewarepath/cohort-middleware/cohort-stats/check-overlap/by-source-id/:sourceid/by-cohort-definition-ids/:cohortdefinitionA/:cohortdefinitionB', (req, res, ctx) => {
+        const { cohortmiddlewarepath } = req.params;
+        const { cohortdefinitionA } = req.params;
+        const { cohortdefinitionB } = req.params;
+        console.log(cohortmiddlewarepath);
+        console.log(cohortdefinitionA);
+        console.log(cohortdefinitionB);
```

Comment:
are these printouts for tthe user?

Reply:
These console.log() statements have been removed now.